offline-virtual-machine.md

```diff
@@ -0,0 +1,74 @@
+# The Offline Virtual Machine user guide
+
+If you require to get hold of non-running virtual machines, you will need to work
+with the OfflineVirtualMachine object. The OfflineVirtualMachine holds the
+template for a running VirtualMachine and holds additional metadata used
+for tracking in the KubeVirt ecosystem.
+
+The OfflineVirtualMachine work with the VirtualMachine, which is a running
+VirtualMachine, that is created upon user request for virtual machine to start.
+
+## When to use it
+
+Whenever you need to manipulate stopped virtual machine (changing/displaying
+configuration).
+
+## How to use it
+
+The OfflineVirtualMachine is designed as a [Kubernetes CRD](https://kubernetes.io/docs/concepts/api-extension/custom-resources/).
+CRD implies that you can use the OfflineVirtualMachine as you would use any
+other Kubernetes object.
+
+### Commandline
+
+Whenever you want to manipulate the OfflineVirtualMachine through the commandline
+you can use the kubectl command. The following are examples demonstrating how
+to do it.
+
+```bash
+# Define an OfflineVirtualMachine:
+kubectl create -f myofflinevm.yaml
+
+# Start an OfflineVirtualMachine:
+kubectl patch offlinevirtualmachine myvm -p \
+    '{"spec":{"running" :"true"}}'
```

Comment:
Please add `--type merge` and also: `"running":true}}"` - notice the boolean true

Reply:
This needs to be addressed!